File: week9_test/uniq_echo.py
#!/usr/bin/python3

import sys

if len(sys.argv) < 2:
	sys.exit(1)
dic = {}
for line in sys.argv:
	if line == sys.argv[0]:
		continue
	if line not in dic:
		print(line,end=' ')
		dic[line] = 1
print()

File: week7_test/uniq_echo.pl
#!/usr/bin/perl -w
use strict;

my %dic = ();
my @a = ();
foreach my $arg(@ARGV){
	next if exists $dic{$arg};
	push @a, $arg;
	$dic{$arg} = 1;		
}
print join ' ', @a, "\n";
